apps/app_voicemail.c

```diff
@@ -4691,10 +4691,10 @@ struct insert_data {
 
 #define STORE_SQL_FMT_CAT "INSERT INTO %s (dir, msgnum, recording, context, callerid, " \
 	"origtime, duration, mailboxuser, mailboxcontext, flag, msg_id, category) " \
-	"VALUES (?,?,?,?,?,?,?,?,?,?,?,?)"
+	"VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?)"
```

Comment:
In the master and 21 branches, there are only 12 named parameters but you've added a 13th so this is going to fail.

Reply:
Yes, I made a mistake, this only applies to 20 branch.